dspace_rest_client/models.py

```diff
@@ -517,4 +517,56 @@ class RelationshipType(AddressableHALResource):
     def __init__(self, api_resource):
         super(RelationshipType, self).__init__(api_resource)
 
+class License(AddressableHALResource):
+    """
+    Specific attributes and functions for licenses
+    """
+    def __init__(self, api_resource=None):
+        super(License, self).__init__(api_resource)
+        self.type = 'clarinlicense'
+        self.name = (api_resource or {}).get('name')
```

Comment:
api_resource = api_resource or {}
and later depend on api_resource to be an initialised dict